src/_Core.scss

```diff
@@ -272,7 +272,7 @@ a.nav-button {
 // override src behavior
 .show button.dropdown-toggle.nav-button,
 button.dropdown-toggle.nav-button:hover {
-  background: unset !important;
```

Comment:
@nickgros , do you know why we need the !important here? I deleted them but if it's going to break something in Portals, please let me know. Thanks.

Reply:
Same here, I don't know why it was there, but it seems like it shouldn't be !important, especially since that breaks the mobile view.